lib/helpers/document.ex

```diff
@@ -1,4 +1,6 @@
 defmodule Xcribe.Helpers.Document do
+  @moduledoc false
+
   alias Xcribe.{Config, ConnParser, Recorder}
 
   defmacro document(conn, opts \\ []) do
```

Comment:
This function is exposed to the user as part of the our public API. Because of that it should be documented.

Reply:
:v: Added !!